src/main/java/org/example/datastructures/queue/ArrayQueue.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package queue;

```

Comment:
This should be 
```java
package org.example.datastructures.queue;
```

Reply:
ok

Reply:
you can merge it now, thanks for your cooperation.